website/project/model.py

```diff
@@ -2935,6 +2935,14 @@ def remove_contributors(self, contributors, auth=None, log=True, save=False):
 
         return all(results)
 
+    def move_contributor(self, user, auth, index, save=False):
```

Comment:
Currently, `Node.manage_contributors` is used to reorder contributors, which also adds a `contributors_reordered` `NodeLog`. This should probably also add one of those logs.

However, `manage_contributors` allows for bulk reordering. Should this endpoint also?